src/NuGet.Core/NuGet.Protocol/HttpSource/HttpRequestMessageExtensions.cs

```diff
@@ -34,7 +34,7 @@ internal static HttpRequestMessage Clone(this HttpRequestMessage request)
                 clone.Headers.TryAddWithoutValidation(header.Key, header.Value);
             }
 
-#if NET5_0
+#if NET5_0 || NET6_0
```

Comment:
can we do NET5_0_OR_HIGHER instead?

Reply:
`NET5_0_OR_GREATER`: https://docs.microsoft.com/en-us/dotnet/standard/frameworks#how-to-specify-a-target-framework

it's a long section in the doc, page down once or twice.

Reply:
Thanks, changed as suggested.